src/components/AvailableOrdersWidget/subcomponents/AvailableOrdersList/AvailableOrdersList.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import { FullOrderERC20 } from "@airswap/typescript";
+
+import useWindowSize from "../../../../hooks/useWindowSize";
+import { AvailableOrdersSortType } from "../../AvailableOrdersWidget";
+import Order from "../Order/Order";
+import {
+  Container,
+  OrdersContainer,
+  Shadow,
+  StyledMyOrdersListSortButtons,
+} from "./AvailableOrdersList.styles";
+
+interface MyOrdersListProps {
+  activeSortType: AvailableOrdersSortType;
+  sortTypeDirection: Record<AvailableOrdersSortType, boolean>;
+  onSortButtonClick: (type: AvailableOrdersSortType) => void;
+  className?: string;
+}
+
+const MyOrdersList: FC<MyOrdersListProps> = ({
+  activeSortType,
+  sortTypeDirection,
+  onSortButtonClick,
+  className,
+}) => {
+  const { t } = useTranslation();
+  const containerRef = useRef<HTMLDivElement>(null);
+  const { width: windowWidth } = useWindowSize();
+  const [invertRate, setInvertRate] = useState(false);
+  const [containerScrollTop, setContainerScrollTop] = useState(0);
+  const [containerWidth, setContainerWidth] = useState(0);
+
+  const handleRateButtonClick = () => {
+    setInvertRate(!invertRate);
+  };
+
+  const handleOnContainerScroll = () => {
+    setContainerScrollTop(containerRef.current?.scrollTop || 0);
+  };
+
+  useEffect(() => {
+    containerRef.current?.addEventListener(
+      "scroll",
+      handleOnContainerScroll.bind(this)
+    );
+
+    return containerRef.current?.removeEventListener(
+      "scroll",
+      handleOnContainerScroll.bind(this)
+    );
+  }, [containerRef]);
+
+  useEffect(() => {
+    setContainerWidth(containerRef.current?.scrollWidth || 0);
+  }, [containerRef, windowWidth]);
```

Comment:
Remove all tooltip related code or implement the tooltip specified in design, up to you!

Implement this:

![Screenshot 2023-03-05 at 22 04 37](https://user-images.githubusercontent.com/86911296/222986038-daff3dfc-ba3b-4c93-a8f4-9dcd118c9bdd.png)

Or just remove (also in `AvailableOrdersList.styles.tsx`):

```suggestion
  const containerRef = useRef<HTMLDivElement>(null);
  const [invertRate, setInvertRate] = useState(false);

  const handleRateButtonClick = () => {
    setInvertRate(!invertRate);
  };
```